changelog/0.34.md

```diff
@@ -0,0 +1,101 @@
+# v0.34.0
+
+Released 2023-11-21
+
+> [!WARNING]
+> **Security Notice(s)**
+> - New curl release (CVE-2023-38545 and CVE-2023-38546)
+>   New Go release (https://github.com/advisories/GHSA-qppj-fm5r-hxr3 and https://github.com/advisories/GHSA-4374-p667-p6c8)
+>   Fix for HTTP/2 Rapid Reset Attack [CVE-2023-44487](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+> - Deploying OpenSearch can now be disabled by setting `opensearch.enabled: false` in `sc-config.yaml`
+> - The number of arguments required to run the `upgrade` command has changed, as you now have to specify the specific cluster, `sc`, `wc`, or `both` to apply to both clusters as the command did previously. The ck8s usage and migration templates have been updated to support and clarify these changes.
+> - The `ck8s init` command now requires a cluster argument which can be either of `<wc|sc|both>`.
+> - Alerts for low remaining free space has changed. There is no longer a **critical** level alert for when free disk space is predicted to run out, but a `warning` level alert remains. Other disk low free disk space alerts have been tweaked.
+> - The default config for storage class name on upcloud has changed from `upcloud-block-storage` to `standard`. This will impact prometheus and log-manager PVCs if they are enabled.
+> - In environments using custom falco rules, falco pods will crash if the overriding rule is disabled by default in this release. Consider enabling the necessary rules using this [config](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/common-config.yaml#L111-L119)
```

Comment:
To make it a bit more clear.
```suggestion
> - [Upstream incubating and sandbox falco rules are now disabled by default](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/common-config.yaml#L111-L119), custom rules overriding these must either be removed or the upstream rules re-enabled, else the falco pods will crash.
```
I'm also thinking that we should perhaps change the link to point to what the release tag would be :thinking:

Reply:
I will do that.